modules/web/js/ballerina/components/editable-text.jsx

```diff
@@ -26,33 +26,46 @@ class EditableText extends React.Component {
             editing: false
         };
 
+        const textBoxPadding = 3;
```

Comment:
Thing is this value affects the svg `<text>` element as well. the `x` value of it should be added the paddingLeft and and borderWith properties so it does not appear as if it jumps. See https://github.com/ballerinalang/composer/pull/1676/files#diff-905b83fb1d50e077c850ce881d193e1eR103